prefixdb_test.go

```diff
@@ -0,0 +1,120 @@
+package db
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func mockDBWithStuff(t *testing.T) DB {
+	db := NewMemDB()
+	// Under "key" prefix
+	require.NoError(t, db.Set(bz("key"), bz("value")))
+	require.NoError(t, db.Set(bz("key1"), bz("value1")))
+	require.NoError(t, db.Set(bz("key2"), bz("value2")))
+	require.NoError(t, db.Set(bz("key3"), bz("value3")))
+	require.NoError(t, db.Set(bz("something"), bz("else")))
+	require.NoError(t, db.Set(bz("k"), bz("val")))
+	require.NoError(t, db.Set(bz("ke"), bz("valu")))
+	require.NoError(t, db.Set(bz("kee"), bz("valuu")))
+	return db
+}
+
+func checkDomain(t *testing.T, itr Iterator, start, end []byte) {
```

Comment:
`checkDomain` is empty function.
WHAT IS `checkDoman'` the rule

Reply:
We removed checkDomain(). However, prefixdb_test.db uses that function. I added empty function to work around.